### ISSUE TITLE
Transfer here "create a simple catalog of anything using markdown"

### ISSUE THREAD

**Issue by rufuspollock** [datopian/makeitmarkdown#19]
## Acceptance

- [x] Transfer https://portaljs.org/blog/create-a-simple-catalog-of-anything-using-markdown over as a post here - source at https://github.com/datopian/portaljs/blob/main/site/content/blog/create-a-simple-catalog-of-anything-using-markdown.md **✅2024-01-03 DONE. See post [here](https://makeitmarkdown.flowershow.app/content/howtos/create-a-simple-catalog-of-anything)**
- [x] x-post to DataHub (at same date as portaljs) - perhaps just a sample with full link to makeitmarkdown **✅2024-01-11 https://datahub.io/blog/create-a-simple-catalog-of-anything-using-markdown**
- [x] Delete from portaljs (with redirect to makeitmarkdown) **❌2024-01-03 WONTFIX. I think we should delete only once Makeitmarkdown is complete, as backup in case anything goes wrong or we need to reference something. Added this to https://github.com/datopian/makeitmarkdown/issues/20 to revisit later**

## Tasks

- [x] Fix images so they show up in the migrated blog post
- [x] x-post to DataHub blog **Elisa doesn't have access to DataHub blog repository. Mohamed has [published the post](https://datahub.io/blog/create-a-simple-catalog-of-anything-using-markdown) but still need to edit so we only have a sample with full link to makeitmarkdown** **✅2024-01-11 https://datahub.io/blog/create-a-simple-catalog-of-anything-using-markdown**

**Comment by elisapaka** [comment]
@rufuspollock Just wanting to check my understanding of the tasks for this issue - please see my questions below.

Questions:
- "How to create a simple of catalog of anything using Markdown files in Obsidian" already exists as a page on the make it markdown site (see [here](https://makeitmarkdown.flowershow.app/content/howtos/create-a-simple-catalog-of-anything)). Is this what you mean by 'transfer over as a post here'?
  - There are some missing images in this post which I was going to fix at a later stage, but I can do that as a matter of priority if we want to cross-post it?
- "x-post to DataHub": my hypothesis is that I should create a post in the general Data Hub Discord channel, containing a sample of the howto page and a link to the full article on the makeitmarkdown site. Is this correct?

**Comment by rufuspollock** [comment]
Ok so we already have this, great!

Though note all the images are missing ... As you note. I would def fix those asap.

(When did this get migrated btw and by whom?)

**Comment by rufuspollock** [comment]
"x-post to DataHub": 

No, I meant post in datahub blog. Ask Mohamed to help you with that.

**Comment by elisapaka** [comment]
> (When did this get migrated btw and by whom?)

I migrated it at the same time as the other posts!

**Comment by elisapaka** [comment]
@rufuspollock I don't have access to the DataHub blog repository - are you able to grant me access?

**Comment by rufuspollock** [comment]
> @rufuspollock I don't have access to the DataHub blog repository - are you able to grant me access?

For now don't worry about this - ask @mohamedsalem401 do do this (and it's minor).

Focus right now is skeleton and getting new stuff written 😊

**Comment by elisapaka** [comment]
> For now don't worry about this - ask @mohamedsalem401 do do this (and it's minor).

Okay! FYI Mohamed has already published the article, I was just going to edit it down to a preview and add the full link to the Makeitmarkdown site.

**Comment by rufuspollock** [comment]
FIXED. Updated acceptance.

> Okay! FYI Mohamed has already published the article, I was just going to edit it down to a preview and add the full link to the Makeitmarkdown site.

@elisapaka great and for future reference you could then update the acceptance (as i have done) with the link to it, namely ...

https://datahub.io/blog/create-a-simple-catalog-of-anything-using-markdown
